project4/styles.css

```diff
@@ -0,0 +1,490 @@
+    text-transform: uppercase;
+}
+
+.button-register:active {
+    color: var(--color-lightgreen);
+    cursor: pointer;
+    display: block;
+    font-family: inherit;
+    font-size: 20px;
+    font-weight: 700;
+    margin: auto;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    text-align: center;
+    text-decoration: underline;
+    text-transform: uppercase;
+}
+
+.conference-photo {
+    -webkit-filter: grayscale(100%);
+    -moz-filter: grayscale(100%);
+    -ms-filter: grayscale(100%);
+    -o-filter: grayscale(100%);
+    filter: grayscale(100%);
+    filter: gray;
+    opacity: 0.7;
+    height: 100%;
+    width: 100%;
+    padding-top: 20px;
+}
+
+.conference-photo:hover {
+    filter: grayscale(0);
+    opacity: 0.6;
+}
+
+
+.day2 {
+    padding-top: 20px;
+}
+
+.email {
+    text-align: center;
+}
+
+.jared-photo,
+.jake-photo,
+.kate-photo {
+    -webkit-filter: drop-shadow(8px 8px 10px black);
+    filter: drop-shadow(8px 8px 10px black);
+    opacity: 0.7;
+    width: 280px;
+    padding-top: 20px;
+}
+
+.jared-photo {
+    padding-top: 20px;
+}
+
+.jake-photo,
+.kate-photo {
+    padding-top: 40px;
+}
+
+.navigation-sst {
+    margin: auto;
+    padding-left: 0;
+}
+
+.navigation-name {
+    margin: auto;
+}
+
+.register {
+    text-align: center;
+}
+
+.speakers {
+    padding-top: 70px;
+}
+
+.schedule,
+.tickets {
+    color: var(--color-lightgreen);
+    padding-top: 70px;
+}
+
+.speakers-name {
+    padding-top: 20px;
+}
+
+.schedule-event,
+.speakers-name {
+    font-size: 20px;
+    font-weight: 600;
+    margin: 0;
+    text-transform: capitalize;
+}
+
+.schedule-time {
+    font-weight: 500;
+    margin: 5px 0;
+    text-transform: uppercase;
+    padding-top: 30px;
+    margin: 0;
+}
+
+.schedule-time2 {
+    padding-top: 0;
+}
+
+.schedule-intro {
+    padding-top: 10px;
+    margin: 0;
+}
+
+.tickets {
+    padding-bottom: 50px;
+}
+
+.tickets-content {
+    text-align: center;
+    padding-bottom: 15px;
+}
+
+.tickets-question {
+    font-size: 24px;
+    font-weight: 500;
+}
+
+.tickets-money {
+    font-size: 20px;
+    font-weight: 500;
+}
+
+.title {
+    padding-top: 80px;
+    padding-bottom: 40px;
+}
+
+/*  ================== CUSTOM STYLES 375px ================== */
+
+@media (min-width:650px) {
+
+    ul {
+        width: 90%;
+        left: 5%;
+        right: 5%;
+        margin: 0;
+        padding-left: 0;
+        position: relative;
+        text-align: left;
+        top: 10px;
+    }
+
+    .navigation-sst {
+        float: right;
+        position: relative;
+        top: 8px;
+        left: -5px;
+    }
+
+    .navigation-name {
+        position: relative;
+        top: 20px;
+        left: 10px;
+    }
+
+}
+
+/*  ================== CUSTOM STYLES 650px ================== */
+
+@media (min-width:720px) {
+
+
+    h1 {
+        font-size: 100px;
+        line-height: 80px;
+        padding-bottom: 50px;
+    }
+
+    h3 {
+        font-size: 50px;
+        line-height: 60px;
+    }
```

Comment:
This, on the other hand, is spot on -- you don't reiterate any existing styles and just write styles for what changes at these sizes. Great work! 👏👏👏